malicious-token-contract-ml-py/src/findings.py

```diff
@@ -55,7 +69,13 @@ def safe_contract_creation(
             }
         )
 
-    def non_malicious_contract_creation(self) -> Finding:
+    def non_malicious_contract_creation(self, chain_id: int) -> Finding:
+        self.metadata["anomaly_score"] = calculate_alert_rate(
+            chain_id,
+            BOT_ID,
+            "NON-MALICIOUS-TOKEN-CONTRACT-CREATION",
+            ScanCountType.TRANSFER_COUNT,
```

Comment:
Could you use `ScanCountType.CONTRACT_CREATION_COUNT` for the denominator?
https://github.com/forta-network/bot-alert-rate/blob/main/python/bot_alert_rate/calculate_alert_rate.py#L12